### DISCUSSION TITLE
手机上怎么搞

### DISCUSSION THREAD

**Discussion by Tuskiy** [share121/inter-knot#2531] [其他]
我下了四五次插件了，还是进不去<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >

**Comment by hutao48** [comment]
不是有教程吗？

**Comment by HKFDBHZN** [comment]
我搞完了，q43397226，我帮你
